### ISSUE TITLE
Language Setting not respected in App

### ISSUE THREAD

**Issue by CalaxDev** [mastodon/mastodon-android#378]
I just joined a Mastodon Server recently and set everything up accordingly, along with my Language Preference on the Instances Websites (set to English)

Now using the Android App however, the language displayed in the App is set to the one my Android OS uses, and there's no way to switch it to English or use the respected language of the Instance Profile Preference.

Is there a way to set the App Language to English other than to switch the entire OS language?

**Comment by grishka** [comment]
Android 13 allows setting per-app languages. If you're using an older Android version, there's currently no way to set the app language independently of the OS setting.

**Comment by CalaxDev** [comment]
@grishka Mastodon Android does not support this feature yet, apparently. It's not available in the list of apps to change language for.

![Screenshot_20221118_104823_Settings](https://user-images.githubusercontent.com/10726046/202672557-4bf844fa-9aad-4e02-854f-0b588e7c15c0.jpg)

**Comment by grishka** [comment]
Ah yes the production version doesn't. It was added in #309 (and I need to build a tool that would automatically update that list)

**Comment by CalaxDev** [comment]
Aah I see - Is there anything I can do to help?
Otherwise I'll just wait for this feature to be released!
(For whatever reason I didn't find this issue when searching before creating this one)

**Comment by grishka** [comment]
No, it's already done and it will be available in the next release.
